pkg/util/subresource.go

```diff
@@ -33,6 +33,24 @@ func GetAgentDetails(h handler) map[string]interface{} {
 	return sub
 }
 
+// GetAgentDetailStrings get all the values for the x-agent-details sub resource as string
+// if a value can not be parsed to a string its value is ignored
+func GetAgentDetailStrings(h handler) map[string]string {
+	details := GetAgentDetails(h)
+	if details == nil {
+		return nil
+	}
+
+	strMap := make(map[string]string)
+
+	for k, v := range details {
```

Comment:
What if something is an int or bool. Should we try to convert that to a string before skipping it?

Reply:
updated to use fmt.Sprint for all values